index.js

```diff
@@ -0,0 +1,6 @@
+const proxy = require('express-http-proxy');
+const app = require('express')();
+
+app.use('/', proxy(process.env.ELASTICSEARCH_CLIENT_URL));
+
+app.listen(8000);
```

Comment:
can this be picked up from `process.env.???` as well?

Reply:
If we pass it as an environment variable from the Dockerfile, yes. It has to be the same port the docker image exposes.